### PR TITLE
add Aloalivn

### PR DESCRIPTION
**Pull request by Robonau**
mentioned by sheep on discord

### CODE REVIEW

File: src/web/mjs/connectors/Aloalivn.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class Aloalivn extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'aloalivn';
+        super.label = 'Aloalivn';
+        this.tags = [ 'manga', 'webtoon', 'english' ];
+        this.url = 'https://aloalivn.com/';
```
Comment (27136773) [inline src/web/mjs/connectors/Aloalivn.mjs]: May remove trailing slash

File: src/web/mjs/connectors/Aloalivn.mjs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        this.queryPages = 'li.blocks-gallery-item source'
```
Comment (Sheepux) [inline src/web/mjs/connectors/Aloalivn.mjs]: Missing semicolon

**Review by Sheepux — CHANGES_REQUESTED** [review]

**Review by 27136773 — CHANGES_REQUESTED** [review]
Required to fix lint errors

**Review by 27136773 — APPROVED** [review]
Looks good

**Comment by 27136773** [comment]
@Sheepux may approve PR if changes are sufficient

**Review by Sheepux — APPROVED** [review]

**Comment by 27136773** [comment]
References: dazedcat19/FMD2#214

**Pull request merged**